app/controllers/chargeback_controller.rb

```diff
@@ -338,6 +389,56 @@ def cb_assign_field_changed
     end
   end
 
+  # Add a new tier at the end
+  def cb_tier_add
+    @edit = session[:edit]
+    i = params[:id]
+    ii = i.to_i
```

Comment:
Bad naming here, actually it is the detail index we want to modify, it comes as a string so we have to converte to integer. I will modify the key name.